.travis/utils.sh

```diff
@@ -135,15 +135,33 @@ function lsr_compare_pythons() {
     $(lsr_get_python_version $1) $2 $(lsr_get_python_version $3)
 }
 
+##
+# lsr_get_system_python
+#
+# Return the system python, or /usr/bin/python3 if nothing
+# else can be found in a standard location.
+function lsr_get_system_python() {
+  local syspython=$(command -pv python3)
+  if [[ -z "$syspython" ]]; then
+    syspython=$(command -pv python)
+  fi
+  if [[ -z "$syspython" ]]; then
+    syspython=$(command -pv python2)
+  fi
+  echo ${syspython:-/usr/bin/python3}
```

Comment:
Sorry, I had to think about this. It seems to be rather odd that /usr/bin/python3 would be a correct choice without it being in PATH. If it is in PATH, it would be set already by line 144. Technically, it should probably fail in this case instead of assuming /usr/bin/python3 to be correct. But maybe this is a wrong assumption. It does not seem to me that this is that important, just wanted to raise it. I am also fine with merging it as-is, since it will probably not hurt. What is your opinion?

Reply:
changed to this:
```
  if [[ -z "$syspython" ]]; then
    lsr_error Could not determine system python path
  fi
  echo $syspython
```